packages/climate/climate.0.1.0/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Declarative command-line parser for OCaml"
+maintainer: ["https://github.com/gridbugs/climate/issues"]
+authors: ["Stephen Sherratt"]
+license: "MIT"
+homepage: "https://github.com/gridbugs/climate"
+bug-reports: "https://github.com/gridbugs/climate/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
The tests fail on all the platforms:

```

#=== ERROR while compiling climate.0.1.0 ======================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/gridbugs/climate/archive/refs/tags/0.1.0.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/climate.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p climate -j 255 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/climate-7-cfe4f0.env
# output-file          ~/.opam/log/climate-7-cfe4f0.out
### output ###
# Error: No rule found for tests/completion_tests/basic/basic.exe
# -> required by alias tests/completion_tests/basic/runtest
# Error: No rule found for tests/completion_tests/fake_git/fake_git.exe
# -> required by alias tests/completion_tests/fake_git/runtest
```

I think the syntax for depending on a binary in dune is not the one you use in your test. It's something like `%{bin:fake_git}` or something like that.

Also note that if you depend on the `exe` explicitely (as oposed to the executable in general), then you might be introducing an implicit dependency to `ocamlopt` being available. It's not a big deal if you do, but in that case you should probably have the corresponding constraint in your opam file (making sure `ocaml-option` is not `bytecode-only` or something like that).

Reply:
These tests are supposed to be part of a different package (`climate_tests`) which I'm not releasing. I've added a `package` field to the `cram` stanzas so these tests don't run as part of `climate`'s tests.